sql/database.sql

```diff
@@ -8085,3 +8085,53 @@ CREATE TABLE `form_clinical_instructions` (
   PRIMARY KEY (`id`)
 )ENGINE=InnoDB;
 -- --------------------------------------------------------
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `ar_activity_sequences`
+--
+CREATE TABLE IF NOT EXISTS `ar_activity_seq` (
+  `pid` int(11) NOT NULL,
+  `encounter` int(11) NOT NULL,
+  `counter` int(11) unsigned NOT NULL,
+  PRIMARY KEY (`pid` , `encounter`)
+) ENGINE=InnoDB;
+-- --------------------------------------------------------
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `claims_sequences`
+--
+CREATE TABLE IF NOT EXISTS `claims_seq` (
+  `patient_id` int(11) NOT NULL,
+  `encounter_id` int(11) NOT NULL,
+  `counter` int(11) unsigned NOT NULL,
+  PRIMARY KEY (`patient_id` , `encounter_id`)
+) ENGINE=InnoDB;
+-- --------------------------------------------------------
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `procedure_answers_sequences`
+--
+CREATE TABLE IF NOT EXISTS `procedure_answers_seq` (
+  `procedure_order_id` bigint(20) NOT NULL DEFAULT '0',
+  `procedure_order_seq` int(11) NOT NULL DEFAULT '0',
+  `question_code` varchar(31) NOT NULL DEFAULT '',
+  `counter` int(11) unsigned NOT NULL,
+  PRIMARY KEY (`procedure_order_id`, `procedure_order_seq`, `question_code`)
+) ENGINE=InnoDB;
+-- --------------------------------------------------------
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `procedure_order_code_sequences`
+--
+CREATE TABLE IF NOT EXISTS `procedure_order_code_seq` (
+  `procedure_order_id` bigint(20) NOT NULL,
+  `counter` int(11) unsigned NOT NULL,
+  PRIMARY KEY (`procedure_order_id`)
+) ENGINE=InnoDB;
+-- --------------------------------------------------------
```

Comment:
place these tables within above file in alphabetical order and mimick formatting of the above tables.
ie. DROP TABLE IF EXISTS etc.